app/js/main.js

```diff
@@ -1020,4 +1067,115 @@ function loadMenu() {
     if (process.platform === "darwin") {
         menuBar.createMacBuiltin("Boats Animator");
     }
+
+}
+
+/**
+ * Lists all of the features that can be set as keyboard shortcuts.
+ */
+var shortcuts = {
+    takePicture: takePicture,
+    undoFrame: undoFrame,
+    audioToggle: function() {
+        audioToggle.checked = !audioToggle.checked;
+    },
+    playPause: function() {
+        btnPlayPause.click();
+    },
+    loopPlayback: function() {
+        btnLoop.click();
+    },
+    liveView: function() {
+        if (totalFrames > 0) {
+            btnLiveView.click();
+        }
+    },
+    confirmOK: function() {
+        btnConfirmOK.click();
+    },
+    confirmCancel: function() {
+        btnConfirmCancel.click();
+    }
+};
+
+/**
+ * Keyboard shortcuts.
+ */
+
+// Shortcuts used in the main window.
+var mainKeys = [
+    // Capture
+    {key: "Enter", active: shortcuts.takePicture},
+    {key: "Delete", active: shortcuts.undoFrame},
+    {key: "Backspace", active: shortcuts.undoFrame},
+    {key: "NumpadMultiply", active: shortcuts.undoFrame},
+    {key: "M", active: shortcuts.audioToggle},
+
+    // Playback
+    {key: "Space", active: shortcuts.playPause},
+    {key: "0", active: shortcuts.playPause},
+    {key: "Numpad0", active: shortcuts.playPause},
+    {key: "MediaPlayPause", active: shortcuts.playPause},
+    {key: "8", active: shortcuts.loopPlayback},
+    {key: "Numpad8", active: shortcuts.loopPlayback},
+
+    // Framereel
+    {key: "L", active: shortcuts.liveView},
+    {key: "3", active: shortcuts.liveView},
+    {key: "Numpad3", active: shortcuts.liveView}
+];
+
+// Shortcuts used in confirm prompts.
+var confirmKeys = [
+    {key: "Enter", active: shortcuts.confirmOK},
+    {key: "Escape", active: shortcuts.confirmCancel},
+    {key: "NumpadDivide", active: shortcuts.confirmCancel}
+];
+
+/**
+ * Choose an array of shortcuts to activate.
+ *
+ * @param {variable} shortcutArray Which array of shortcuts should be enabled.
+ *                                 eg mainKeys or confirmKeys.
+ */
+function addShortcuts(shortcutArray) {
```

Comment:
Can we move this add/remove mechanism to a separate script (`shortcuts.js`) and `require()` it into here (`file.js` would be a good model to follow)?